libandroid-navigation/src/main/java/org/maplibre/navigation/android/navigation/v5/models/StepManeuver.kt

```diff
@@ -0,0 +1,233 @@
+package org.maplibre.navigation.android.navigation.v5.models
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import org.maplibre.geojson.Point
+import org.maplibre.navigation.android.navigation.v5.models.serializer.PointSerializer
+
+/**
+ * Gives maneuver information about one [LegStep].
+ *
+ * @since 1.0.0
+ */
+@Suppress("unused")
+@Serializable
+data class StepManeuver(
+
+    /**
+     * A [Point] representing this intersection location.
+     *
+     * @since 3.0.0
+     */
+    @Serializable(with = PointSerializer::class)
+    val location: Point,
+
+    /**
+     * Number between 0 and 360 indicating the clockwise angle from true north to the direction of
+     * travel right before the maneuver.
+     *
+     * @since 1.0.0
+     */
+    @SerialName("bearing_before")
+    val bearingBefore: Double,
+
+    /**
+     * Number between 0 and 360 indicating the clockwise angle from true north to the direction of
+     * travel right after the maneuver.
+     *
+     * @since 1.0.0
+     */
+    @SerialName("bearing_after")
+    val bearingAfter: Double,
+
+    /**
+     * A human-readable instruction of how to execute the returned maneuver. This String is built
+     * using OSRM-Text-Instructions and can be further customized inside either the Mapbox Navigation
+     * SDK for Android or using the OSRM-Text-Instructions.java project in Project-OSRM.
+     *
+     * @see [Navigation SDK](https://github.com/mapbox/mapbox-navigation-android)
+     * @see [OSRM-Text-Instructions.java](https://github.com/Project-OSRM/osrm-text-instructions.java)
+     *
+     * @since 1.0.0
+     */
+    val instruction: String?,
+
+    /**
+     * This indicates the type of maneuver.
+     *
+     * @since 1.0.0
+     */
+    val type: Type?,
+
+    /**
+     * This indicates the mode of the maneuver. If type is of turn, the modifier indicates the
+     * change in direction accomplished through the turn. If the type is of depart/arrive, the
+     * modifier indicates the position of waypoint from the current direction of travel.
+     *
+     * @since 1.0.0
+     */
+    val modifier: String?,
```

Comment:
Would it be reasonable to use the `ManeuverModifier.Type` instead of `String` here?

Reply:
Done ✅